app/models/protocol.rb

```diff
@@ -460,35 +460,11 @@ def any_service_requests_to_display?
     return self.service_requests.detect { |sr| !['first_draft'].include?(sr.status) }
   end
 
-  def has_per_patient_per_visit? current_request, portal
+  def has_line_items_of_type? current_request, portal, one_time_fee = false
```

Comment:
having "of_type" in the name implies that I should see a type in the parameters of a call to the function